src/Orckestra.Composer.Search/Services/BaseSearchViewService.cs

```diff
@@ -540,6 +584,12 @@ protected virtual async Task<ProductSearchResultsViewModel> SearchAsync(TParam p
                 SearchResult = searchResult
             };
 
+            if (param.Criteria.SelectedFacets != null && 
+                param.Criteria.SelectedFacets.Any(s => !string.IsNullOrWhiteSpace(s.Name) && s.Name.StartsWith(SearchConfiguration.CategoryFacetFiledNamePrefix)))
```

Comment:
Can be simplified

s => !string.IsNullOrWhiteSpace(s.Name) && s.Name.StartsWith(SearchConfiguration.CategoryFacetFiledNamePrefix)

s => s.Name?.StartsWith(SearchConfiguration.CategoryFacetFiledNamePrefix) ?? false